src/Dropdown/Dropdown.tsx

```diff
@@ -198,23 +199,16 @@ const StyledSelect = styled.select<UsesOutline>`
   ${({ outlined }) =>
     outlined &&
     `
-  border: 2px solid ${theme.colors.outline};
-  border-radius: 8px;
-  padding: 16px 12px;
-  box-sizing: border-box;
-  height: auto;
-  &:hover,
-  &:focus-within {
-    border: 2px solid ${theme.colors.outline};
-  }
-`}
-  ${({ value }) =>
-    value &&
-    value != '' &&
-    `
-    border: 2px solid ${theme.colors.outline};
-`}
```

Comment:
I've removed this, I think it was not intended, there is a border when the component is controlled